guacamole_image/Dockerfile

```diff
@@ -7,7 +7,7 @@ ENV GUACAMOLE_HOME /opt/guacamole/.guacamole
 ENV GUACAMOLE_PROPERTIES guacamole.properties
 
 # ARG of the JAR version
-ARG JAR_VERSION=1.0.0
```

Comment:
I think its irrelevant because we inject this argument when we build the Image

Reply:
I have tested the build and forget to rollback. fixed